components/post_markdown/system_message_helpers.jsx

```diff
@@ -44,6 +44,18 @@ function renderJoinChannelMessage(post) {
     );
 }
 
+function renderGuestJoinChannelMessage(post) {
+    const username = renderUsername(post.props.username);
+
+    return (
+        <FormattedMessage
+            id='api.channel.guest_join_channel.post_and_forget'
+            defaultMessage='{username} joined the channel as guets.'
```

Comment:
Please update to: "{username} joined the channel as a guest."  (note spelling on guest and adding a "a")